controllers/kyma_controller.go

```diff
@@ -429,3 +430,12 @@ func (r *KymaReconciler) deleteModule(ctx context.Context, moduleStatus *v1beta1
 	manifest.SetName(moduleStatus.Manifest.GetName())
 	return r.Delete(ctx, &manifest, &client.DeleteOptions{})
 }
+
+// RecordKymaStatusMetrics updates prometheus metrics defined to track changes to the Kyma status
+func (r *KymaReconciler) RecordKymaStatusMetrics(kyma *v1beta1.Kyma) {
+	//TODO: where to take "instanceId" and "shoot" from?
```

Comment:
they come from labels in the runtime, please see our managed clusters and observe what KEB is introducing there.

Reply:
@lindnerby is on checking if these labels are automatically added to any metrics. In that case we don't have to do anything. For now I removed these from the code.

Reply:
As discussed, we can get these from the Kyma CR.